fp.js

```diff
@@ -31,4 +46,6 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+//var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+ //This is my answer to #3//
+var s_people = people.forEach(person =>person.name.match("s"))
```

Comment:
Remember that `forEach` returns `undefined`. If you want to create a new collection using `forEach`, you need to push something into a new array. So the answer here would look something like..

```js
var people = { ... } // original data in here
let peopleWhoseNamesHaveTheLetterS = []
people.forEach(person => {
  peopleWhosNamesHaveTheLetterS.push(person);
}
```